### PR TITLE
Fibonacci iterative and recursive

### PR DESCRIPTION
**Pull request by kondorski**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: fibonacci
🏅 Granted 6 XP for @kondorski

**Comment by ziobron** [comment]
⏱ 3 XP extra for @kondorski for on-time delivery

File: homework/fibonacci/fibonacci.hpp [hunk 1]
```diff
@@ -1,11 +1,28 @@
...
-
-int fibonacci_recursive(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
-}
+#pragma once
+
+int fibonacci_iterative(int sequence) {
+    int t1 = 0;
+    int t2 = 1;
```
Comment (nauka-programowania-MA) [inline homework/fibonacci/fibonacci.hpp:5]: What are 't1' and `t2` ? Variable should always represent real operation.
Like `result` `divider` `multiplier` `error_code` etc...

File: homework/fibonacci/fibonacci.hpp [hunk 2]
```diff
@@ -1,11 +1,28 @@
...
+    int next_term;
+    
+    if(sequence < 2){
+        return sequence;
+    }
+    else {
```
Comment (nauka-programowania-MA) [inline homework/fibonacci/fibonacci.hpp:11]: If you return variable here, you don't need to use `else`

File: homework/fibonacci/fibonacci.hpp [hunk 3]
```diff
@@ -1,11 +1,28 @@
...
+        return next_term;
+    }
+}
+
+int fibonacci_recursive(int sequence) {
+    if(sequence < 2){
+        return sequence;
+    }
+    else{
+        return fibonacci_recursive(sequence-1) + fibonacci_recursive(sequence-2);
```
Comment (nauka-programowania-MA) [inline homework/fibonacci/fibonacci.hpp:26]: If you return variable here, you don't need to use `else `

Comment (nauka-programowania-MA) [inline homework/fibonacci/fibonacci.hpp:26]: Add space between operations like '+' '-' '!=' '==' etc...

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**